pandas/tests/test_resample.py

```diff
@@ -2165,6 +2165,13 @@ def test_resample_datetime_values(self):
         res = df['timestamp'].resample('2D').first()
         tm.assert_series_equal(res, exp)
 
+    def test_resample_quantile(self):
```

Comment:
can you test on timedelta / period as well. you might want to move to the Base class to make this easier.

Reply:
This example mentioned in the description of issue doesn't work for timedelta:

```
s = self.create_series()
expected = s.resample('W').agg(lambda x: x.quantile(0.75))
```

trace:
```
.../pandas/pandas/tseries/offsets.py in nanos(self)
    399     @property
    400     def nanos(self):
--> 401         raise ValueError("{name} is a non-fixed frequency".format(name=self))
    402
    403

ValueError: <Week: weekday=6> is a non-fixed frequency
```

@jreback 
Let me know please if it's correct.